doc/pattern_tools/gen_pattern.py

```diff
@@ -96,8 +166,10 @@ def main():
                         dest="page_width", type=float)
     parser.add_argument("-h", "--page_height", help="page height in units", default=argparse.SUPPRESS, action="store",
                         dest="page_height", type=float)
-    parser.add_argument("-a", "--page_size", help="page size, superseded if -h and -w are set", default="A4", action="store",
-                        dest="page_size", choices=["A0", "A1", "A2", "A3", "A4", "A5"])
+    parser.add_argument("-a", "--page_size", help="page size, superseded if -h and -w are set", default="A4",
+                        action="store", dest="page_size", choices=["A0", "A1", "A2", "A3", "A4", "A5"])
+    parser.add_argument("-m", "--markers", help="list of cells with markers for the radon checkerboard",
```

Comment:
I think marker format should be noted here too, something like 
>marker coordinates as list of numbers: `-m 1 2 3 4` means markers in cells [1, 2] and [3, 4]

Reply:
added